app/assets/javascripts/discourse/app/components/sidebar/user/custom-sections.hbs

```diff
@@ -5,11 +5,30 @@
       @headerLinkText={{section.decoratedTitle}}
       @collapsable={{true}}
       @headerActions={{section.headerActions}}
-      @headerActionsIcon="pencil-alt"
+      @headerActionsIcon={{section.headerActionIcon}}
       @class={{section.dragCss}}
     >
       {{#each section.links as |link|}}
-        {{#if link.external}}
+        {{#if section.section_type}}
```

Comment:
Is there a way we can combine this with the `else`? I feel like custom sections should have links all behaving the same way.